apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js

```diff
@@ -69,16 +73,16 @@ const ReviewApplication = ({ issue, requestIndex }) => {
   const application = {
     user: {
       login: request.user.login,
-      name: request.user.login,
-      avatar: request.user.avatarUrl,
+      name: request.user.name,
+      avatarUrl: request.user.avatarUrl,
       url: request.user.url
     },
     workplan: request.workplan,
     hours: request.hours,
     eta: (request.eta === '-') ? request.eta : (new Date(request.eta)).toLocaleDateString(),
     applicationDate: request.applicationDate
   }
-
+  
```

Comment:
Please configure your editor to strip trailing whitespace when you save a file 🙂

Reply:
It actually is configured that way, although it got confused and stopped removing whitespaces. It needed restarting to get that functionality back. Thank you, VSCode...